app/uk/gov/hmrc/helptosavefrontend/config/frontendAppConfig.scala

```diff
@@ -35,7 +35,7 @@ object FrontendAppConfig extends AppConfig with ServicesConfig {
   private val contactHost = configuration.getString(s"contact-frontend.host").getOrElse("")
   private val contactFormServiceIdentifier = "MyService"
 
-  val HtsUserDetailsUrl = getConfString("help-to-save-user-details.url", "")
+  val HtsConfirmDetailsUrl = getConfString("help-to-save-confirm-details.url", "")
```

Comment:
the DEV evnironment wants an absolute url  and the QA envorinment want a relative URL